lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -4949,6 +5470,64 @@ impl FunctionCodeGenerator<CodegenError> for X64FunctionCode {
                 }
             }
 
+            Operator::I64TruncSSatF64 => {
+                let loc =
+                    get_location_released(a, &mut self.machine, self.value_stack.pop().unwrap());
+                let ret = self.machine.acquire_locations(
+                    a,
+                    &[(WpType::I64, MachineValue::WasmStack(self.value_stack.len()))],
+                    false,
+                )[0];
+                self.value_stack.push(ret);
+
+                let tmp_out = self.machine.acquire_temp_gpr().unwrap();
+                let tmp_in = self.machine.acquire_temp_xmm().unwrap();
+
+                Self::emit_relaxed_binop(
+                    a,
+                    &mut self.machine,
+                    Assembler::emit_mov,
+                    Size::S64,
+                    loc,
+                    Location::XMM(tmp_in),
+                );
+                Self::emit_f64_int_conv_check_sat(
+                    a,
+                    &mut self.machine,
+                    tmp_in,
+                    -9223372036854777856.0,
+                    9223372036854775808.0,
```

Comment:
The bottom number is the same but the top number is slightly different:

from
```
                    -9223373136366403584.0,
                    9223372036854775808.0,
```

names would make that more obvious

Reply:
`GEF64_LT_I64_MIN` and `LEF64_GT_I64_MAX`.